Signal/src/ViewControllers/MessagesViewController.m

```diff
@@ -3158,7 +3162,7 @@ - (void)endRecordingVoiceMemo
     self.audioRecorder = nil;
 
     NSString *filename = [NSLocalizedString(@"VOICE_MESSAGE_FILE_NAME", @"Filename for voice messages.")
-        stringByAppendingPathExtension:[MIMETypeUtil fileExtensionForUTIType:(NSString *)kUTTypeMPEG4Audio]];
+        stringByAppendingPathExtension:@".m4a"];
```

Comment:
Actually, it occurs to me now that it's not sufficient to do this fix - we should stop using `fileExtensionForUTIType:` entirely.  Incoming .m4a files without a filename would be wrongly saved as .mp4 which, when shared, will be wrongly treated as video.  I'll do this now if possible.